server/superdesk/io/rfc822.py

```diff
@@ -23,11 +24,25 @@
 from superdesk.errors import IngestEmailError
 from bs4 import BeautifulSoup, Comment, Doctype
 import re
+from eve.utils import ParsedRequest
 
 
 logger = logging.getLogger(__name__)
 
 
+class UserNotRegisteredException(Exception):
+    pass
+
+
+def get_user_by_email(from_field):
+    email_address = re.compile('^.*<(.*)>$').findall(from_field)[0]
```

Comment:
oops, i was going to move the compilation of the regex outside of the function, will fix it tomorrow